lib/livebook/notebook/explore/kino/custom_kinos.livemd

```diff
@@ -92,8 +123,8 @@ defmodule Kino.Leaflet do
     Kino.JS.Live.new(__MODULE__, {normalize_location(center), zoom})
   end
 
-  def add_marker(widget, location) do
-    Kino.JS.Live.cast(widget, {:add_marker, normalize_location(location)})
+  def add_marker(kino, location) do
```

Comment:
We should match this in to **counter** example below, though I'm not sure if `kino` is more clear than `widget` in this context. Alternatively we could just call it `server`?

Reply:
I think they are `kino` "objects" from the user point of view but the API will message the server hidden from it. In any case though, I want to consolidate on using the word `kino` instead of `widget`, because we have been using both, so I though it was a good to pick one and `kino` is more Livebook specific. :)

Reply:
Alright!